app/Word.php

```diff
@@ -16,4 +18,14 @@ public function choices()
     {
         return $this->hasMany(Choice::class);
     }
+
+    public function newRecord($attributes){
+        $record = $this->create([
+            'category_id' => $attributes['category_id'],
+            'text' => $attributes['text'],
+        ]);
+        $attributes['word_id'] = $record->id;
+        $choices = new Choice;
+        $choices->newRecord($attributes);
```

Comment:
You are passing the attributes array, but there are unnecessary part which is `category_id` and `text`, it is not related to choices anymore. You can use laravel collection to filter the array and just pass what is important.